src/hooks/useClaimableWalletToken/useClaimableWalletToken.ts

```diff
@@ -9,30 +9,40 @@ import { getProvider } from '../../services/provider'
 import useCacheBreak from '../useCacheBreak'
 import { UseClaimableWalletTokenProps, UseClaimableWalletTokenReturnType } from './types'
 
+// const supplyControllerAddress = '0xF8cF66BbF7fe152b8177B61855E8be9a6279C8A1' //test polygon
 const supplyControllerAddress = '0xc53af25f831f31ad6256a742b3f0905bc214a430'
+const WALLET_STAKING_ADDR = '0x47Cd7E91C3CBaAF266369fe8518345fc4FC12935'
 const supplyControllerInterface = new Interface(WALLETSupplyControllerABI)
+const NETWORK_NAME = 'ethereum'
 
 const useClaimableWalletToken = ({
-  useConstants,
+  relayerURL,
+  useRelayerData,
   accountId,
   network,
   addRequest,
   totalLifetimeRewards,
   walletUsdPrice
 }: UseClaimableWalletTokenProps): UseClaimableWalletTokenReturnType => {
-  const { constants } = useConstants()
-  const provider = useMemo(() => getProvider('ethereum'), [])
+  const {cacheBreak: relayerCacheBreak} = useCacheBreak()
+  const urlIdentityRewards = relayerURL
+    ? `${relayerURL}/wallet-token/rewards/${accountId}?cacheBreak=${relayerCacheBreak}`
+    : null
+
+  const rewardsData = useRelayerData({ url: urlIdentityRewards })
+  const claimableRewardsData = rewardsData?.data?.claimableRewardsData
+  const provider = useMemo(() => getProvider(NETWORK_NAME), [])
   const supplyController = useMemo(
     () => new Contract(supplyControllerAddress, WALLETSupplyControllerABI, provider),
     [provider]
   )
   const initialClaimableEntry = useMemo(() => {
-    if (!constants?.WALLETInitialClaimableRewards) {
+    if (!claimableRewardsData) {
       return null
     }
 
-    return constants.WALLETInitialClaimableRewards.find((x) => x.addr === accountId)
-  }, [accountId, constants?.WALLETInitialClaimableRewards])
+    return claimableRewardsData //.find((x) => x.addr === accountId)
```

Comment:
I see no need to have `claimableRewardsData` then. Simply:

```
const claimableRewardsData = rewardsData?.data?.claimableRewardsData || null
```

Will be the same now.